lxd/db/storage_volumes.go

```diff
@@ -246,18 +246,29 @@ func (c *Cluster) storagePoolVolumeGetType(project string, volumeName string, vo
 		return -1, nil, err
 	}
 
+	volumeContentType, err := c.GetStorageVolumeContentType(volumeID)
```

Comment:
It seems `GetStorageVolumeContentType` is not used outside the `lxd/db` package, so it should probably be unexported (e.g. `getStorageVolumeContentType`).

Additionally, it's a bit unfortunate that it starts a transaction just to fetch that one column. Ideally the whole code in the calling `storagePoolVolumeGetType` method should be changed to run only one transaction.

That might be outside the scope of this PR though, and be done separately if we wish to spend time on that.